views/editor/projects.html

```diff
@@ -27,8 +27,10 @@
         {% endfor %}
         >
           <td class="project-title">
-            {{ project.title }}
-            <span class="project-information"></span>
+            <a target="_self" href="/{{locale}}/user/{{username}}/{{project.id}}/{{queryString}}"> 
```

Comment:
You are right! I am really sorry about that @gideonthomas. I have made changes and it should work now.